### DISCUSSION TITLE
Terminal error

### DISCUSSION THREAD

**Discussion by theikid** [louislam/dockge#658] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I'm running Dockge inside truenas scale having installed it from the app catalog.  I create a first stack and want to access the bash of this container without luck (message below). 

### 📝 Error Message(s) or Log

"OCI runtime exec failed: exec failed: unable to start container process: exec: "bash": executable file not found in $PATH: unknown"

### 🐻 Dockge Version

1.4.2

### 💻 Operating System and Arch

Truenas Scale

### 🌐 Browser

Safari

### 🐋 Docker Version

27.1.1

**Comment by luckman212** [comment]
Try the **Switch to sh** button?

<img width="163" alt="image" src="https://github.com/user-attachments/assets/7bd1b91b-eeea-4d4e-a013-5a0ede8c4ef2" />

**Reply by Patryk122PL** [reply]
> Try the **Switch to sh** button?
> 
> <img alt="image" width="163" src="https://private-user-images.githubusercontent.com/1992842/438248965-7bd1b91b-eeea-4d4e-a013-5a0ede8c4ef2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDYxMDc1MzUsIm5iZiI6MTc0NjEwNzIzNSwicGF0aCI6Ii8xOTkyODQyLzQzODI0ODk2NS03YmQxYjkxYi1lZWVhLTRkNGUtYTAxMy01YTBlZGU4YzRlZjIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDUwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA1MDFUMTM0NzE1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9N2FiOGEzZmMxYjkwM2Y1ZmIxNDE0MjExZTYyMmRmNjkzZTMyYzNkOTZiNjQ4NDg1ODZlMDZiYTNjNjAyYmI4MiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.bi4o9YDpjzgnstE5bs9Y2WKkRPsUnRaIZ9kscYl3AOo">

Is there a way to set the default terminal? I have some containers that work with bash shell and others that are sh only. It's a small nitpick but also kind of regular annoyance to have to click this button after every container restart.

**Reply by luckman212** [reply]
I agree this would be "nice to have" (setting a default shell)
